Tests/LibWeb/Text/input/hit_testing/overflow-scroll.html

```diff
@@ -0,0 +1,49 @@
+<script src="../include.js"></script>
+<style type="text/css">
+    #container {
+        border: 1px solid black;
+        height: 200px;
+        overflow: scroll;
+    }
+
+    p {
+        margin: 0;
+    }
+
+    p:hover {
+        background-color: yellow;
+    }
+</style>
+
+<body>
+    <div id=container>
+        <p id="line-1">Line 1</p>
+        <p id="line-2">Line 2</p>
+        <p id="line-3">Line 3</p>
+        <p id="line-4">Line 4</p>
+        <p id="line-5">Line 5</p>
+        <p id="line-6">Line 6</p>
+        <p id="line-7">Line 7</p>
+        <span id="line-8">Line 8</span>
```

Comment:
span is used for this line to make sure hit-testing works for inline-paintables.